test/src/sender/main.cpp

```diff
@@ -23,119 +23,164 @@
+    }
 };
 
 
 int main(){
 
-  auto start = hclock::now(); //to measure simulation execution time
+    auto start = hclock::now(); //to measure simulation execution time
 
-/*************** Loggers *******************/
-  static std::ofstream out_data("../test/data/sender/sender_test_output.txt");
+    /*************** Loggers *******************/
+    static std::ofstream out_data(
+        "../test/data/sender/sender_test_output.txt");
     struct oss_sink_provider{
-        static std::ostream& sink(){          
+        static std::ostream& sink(){
             return out_data;
         }
     };
 
-using info=cadmium::logger::logger<cadmium::logger::logger_info, cadmium::dynamic::logger::formatter<TIME>, oss_sink_provider>;
-using debug=cadmium::logger::logger<cadmium::logger::logger_debug, cadmium::dynamic::logger::formatter<TIME>, oss_sink_provider>;
-using state=cadmium::logger::logger<cadmium::logger::logger_state, cadmium::dynamic::logger::formatter<TIME>, oss_sink_provider>;
-using log_messages=cadmium::logger::logger<cadmium::logger::logger_messages, cadmium::dynamic::logger::formatter<TIME>, oss_sink_provider>;
-using routing=cadmium::logger::logger<cadmium::logger::logger_message_routing, cadmium::dynamic::logger::formatter<TIME>, oss_sink_provider>;
-using global_time=cadmium::logger::logger<cadmium::logger::logger_global_time, cadmium::dynamic::logger::formatter<TIME>, oss_sink_provider>;
-using local_time=cadmium::logger::logger<cadmium::logger::logger_local_time, cadmium::dynamic::logger::formatter<TIME>, oss_sink_provider>;
-using log_all=cadmium::logger::multilogger<info, debug, state, log_messages, routing, global_time, local_time>;
-
-using logger_top=cadmium::logger::multilogger<log_messages, global_time>;
-
-
-/*******************************************/
-
-
-
-/********************************************/
-/****** APPLICATION GENERATOR *******************/
-/********************************************/
-string input_data_control = "../test/data/sender/sender_input_test_control_In.txt";
-const char * i_input_data_control = input_data_control.c_str();
-
-std::shared_ptr<cadmium::dynamic::modeling::model> generator_con = cadmium::dynamic::translate::make_dynamic_atomic_model<ApplicationGen, TIME, const char* >("generator_con" , std::move(i_input_data_control));
-
-string input_data_ack = "../test/data/sender/sender_input_test_ack_In.txt";
-const char * i_input_data_ack = input_data_ack.c_str();
-
-std::shared_ptr<cadmium::dynamic::modeling::model> generator_ack = cadmium::dynamic::translate::make_dynamic_atomic_model<ApplicationGen, TIME, const char* >("generator_ack" , std::move(i_input_data_ack));
-
-
-/********************************************/
-/****** SENDER *******************/
-/********************************************/
-
-std::shared_ptr<cadmium::dynamic::modeling::model> sender1 = cadmium::dynamic::translate::make_dynamic_atomic_model<Sender, TIME>("sender1");
+    using
```

Comment:
1) We would suggest to follow the below format, 
    using info = cadmium::logger::logger<
        cadmium::logger::logger_info, 
        cadmium::dynamic::logger::formatter<TIME>, 
        oss_sink_provider
    >;

2) a space is required before and after =

Reply:
For 1 comment i did like that so it can be understood that its in continuation

Reply:
for 2 it has been solved

Reply:
for 1) works for me and 2) thanks for the correction!